sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/autorest.md

```diff
@@ -9,7 +9,7 @@ Run `generate.ps1` or `dotnet msbuild /t:GenerateCode` to generate code.
 public-clients: true
 title: DeviceUpdate
 
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b9b91929c304f8fb44002267b6c98d9fb9dde014/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2022-07-01-preview/deviceupdate.json
+input-file: https://raw.githubusercontent.com/dpokluda/azure-rest-api-specs/bbf13b0366e5863ce65f84d588ed5681326af0cb/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json
```

Comment:
We need to use the swagger file in swagger public repo: azure/azure-rest-api-spec

Reply:
Fixed.